rb_tree.py

```diff
@@ -21,11 +23,11 @@ def __repr__(self):
 
     def __iter__(self):
         if self.left.color != NIL:
+            print('left ({})'.format(str(self.value)))
             yield from self.left.__iter__()
 
-        yield self.value
```

Comment:
This will be a bug - we should not remove this line at all.

Reply:
Ok!